dev/README_RELEASE_AIRFLOW.md

```diff
@@ -477,8 +496,12 @@ To do this we need to
     twine upload -r pypitest dist/*
     ```
 
-- Verify that the test package looks good by downloading it and installing it into a virtual environment. The package download link is available at:
-https://test.pypi.org/project/apache-airflow/#files
+- Copy the package tar.gz download link available at https://test.pypi.org/project/apache-airflow/#files. Verify that the test package looks good by
+installing it with the constraint file into a new virtual environment. (adapt the constraint file python version to your virtual env setup)
+
+    ```shell script
+    pip install https://test-files.pythonhosted.org/packages/98/2c/82009f7760f341cc4e12c44da657c906415130ca60cd201d05e9b38caa38/apache-airflow-2.2.1rc1.tar.gz --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.2.1rc1/constraints-3.8.txt
```

Comment:
Maybe we can do it myuch simpler:

```
python3 -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ apache-airflow-2.2.1rc1.tar.gz
```

This will use both test pypi (to install RC) and regular PyPI (for all other packages).

Reply:
Yes, I believe this is equivalent to this: (Tried it locally it works)
```
pip install -i https://test.pypi.org/simple/ apache-airflow==2.5.1rc1
```

I have an error when running your command. What constraint file will be used here ? Do we explicitly pass the constraint file so we verify that dependencies can be resolve with new constrainst ?

Reply:
Yes. we need to add same constraints (I just did not add them in my example).

Hmm. I always had a trouble when installing only from "test.pypi" because some dependent packages were not installed there (not everyone uses test.pypi to test their packages). So once you add constraints, I think without the --extra-index-url things might start failing because specific versions of dependencies in the constraints might be missing.